Dockerfile

```
@@ -20,6 +20,9 @@ RUN chmod 755 /sbin/entrypoint.sh
 
 EXPOSE 3142/tcp
 
+HEALTHCHECK --interval=10s --timeout=2s --retries=3 \
+    CMD curl -f http://localhost:3142/acng-report.html || exit 1
```

Consequently we have,

```suggestion
    CMD wget -qO - http://localhost:3142/acng-report.html || exit 1
```